sdk/state/integration/helpers_test.go

```diff
@@ -178,7 +216,53 @@ func retry(maxAttempts int, f func() error) (err error) {
 	return err
 }
 
-func fund(client horizonclient.ClientInterface, account *keypair.FromAddress, startingBalance int64) error {
+func fundAsset(client horizonclient.ClientInterface, asset txnbuild.Asset, amount int64, accountKP *keypair.Full, distributorKP *keypair.Full) error {
+	distributor, err := client.AccountDetail(horizonclient.AccountRequest{AccountID: distributorKP.Address()})
+	if err != nil {
+		return err
+	}
+
+	ops := []txnbuild.Operation{}
+	if !asset.IsNative() {
+		ops = append(ops, &txnbuild.ChangeTrust{
+			SourceAccount: accountKP.Address(),
+			Line:          asset,
+			Limit:         "5000",
+		})
+	}
+	ops = append(ops, &txnbuild.Payment{
+		Destination: accountKP.Address(),
+		Amount:      stellarAmount.StringFromInt64(amount),
+		Asset:       asset,
+	})
+	tx, err := txnbuild.NewTransaction(txnbuild.TransactionParams{
+		SourceAccount:        &distributor,
+		IncrementSequenceNum: true,
+		BaseFee:              txnbuild.MinBaseFee,
+		Timebounds:           txnbuild.NewTimeout(300),
+		Operations:           ops,
+	})
+	if err != nil {
+		return err
+	}
+	if !asset.IsNative() {
+		tx, err = tx.Sign(networkPassphrase, accountKP)
+		if err != nil {
+			return err
+		}
+	}
+	tx, err = tx.Sign(networkPassphrase, distributorKP)
+	if err != nil {
+		return err
+	}
+	_, err = client.SubmitTransaction(tx)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func createAccount(client horizonclient.ClientInterface, account *keypair.FromAddress, startingBalance int64) error {
```

Comment:
changing the name from `fund` to `createAccount`.  That's really what this method is doing, and to differentiate from the `fundAsset` method

Reply:
good idea to rename! 💯